Library/Util.ts

```diff
@@ -158,6 +158,8 @@ class Util {
                 if (propertyType !== "string") {
                     if (property != null && typeof property.toString === "function") {
                         property = property.toString();
+                    } else if (property === null || propertyType === "undefined") {
+                        property = "";
```

Comment:
Could you comment on the scenario where this is useful? If the value for the custom dimension is missing, why incur extra billing cost for bytes ingested with the dimension name?

Reply:
This is useful when you have a manager or operations person looking through logs, who is not necessarily able/willing to access the codebase, who's going to ask the question "why isn't this being logged?" when it actually is.

You have a good point regarding billing cost - I'm not the one footing the bill, here 😉